django_cloud_deploy/tool_requirements.py

```diff
@@ -157,14 +157,16 @@ def check(cls):
             subprocess.check_call(command, stdout=subprocess.DEVNULL,
                                   stderr=subprocess.DEVNULL)
         except subprocess.CalledProcessError:
-            # TODO: Handle for multiple OS's
-            # TODO: Check output for error message given when user has
-            # not ran the command
-            msg = ('Docker is installed but not correctly set up.'
-                   'Use the following command to fix it: \n'
-                   '$ sudo groupadd docker\n'
-                   '$ sudo usermod -a -G docker $USER\n'
-                   'Then log out/log back in')
+            if sys.platform.startswith('linux'):
```

Comment:
You could also do `getent group docker` and see if the user's name is in it. Not sure if that is overkill or not.

Reply:
Is there any gain from doing that? Thats a subprocess command as well right?

Reply:
You are recommending that the user do because you got an error from docker. But you don't actually know that it is related to group membership. `getent` would allow you to check.